pcdsdevices/lasers/elliptec.py

```diff
@@ -3,56 +3,137 @@
 """
 
 
-from ophyd import Component as Cpt
+from ophyd import FormattedComponent as FCpt
 from ophyd import Device
 from ophyd.signal import EpicsSignal, EpicsSignalRO
 
 
 class EllBase(Device):
     """
     Base class for Elliptec stages.
+    """
+
+    set_position = FCpt(EpicsSignal, '{prefix}:M{self._channel}:CURPOS',
```

Comment:
Not important to change, but you can omit all `self.` in the formatted component strings:
`'{prefix}:M{_channel}:CURPOS'`

Reply:
Darn it @ZLLentz 😆